### PR TITLE
979 default naming pattern

### PR DESCRIPTION
**Pull request by abdelr**
979 default naming pattern

### CODE REVIEW

File: src/OSPSuite.Presentation/Presenters/Importer/ImportConfirmationPresenter.cs [hunk 1]
```diff
@@ -13,6 +14,9 @@ public class ImportConfirmationPresenter : AbstractPresenter<IImportConfirmation
       private readonly IDataRepositoryChartPresenter _chartPresenter;
       private readonly IDataRepositoryDataPresenter _dataPresenter;
       private string _lastNamingPattern = "";
+      private DataRepository _dataRepository;
+      private List<string> _conventions;
+      private IEnumerable<string> _keys;
```
Comment (msevestre) [inline src/OSPSuite.Presentation/Presenters/Importer/ImportConfirmationPresenter.cs:19]: Can it be a IReadOnly?

**Review by msevestre — COMMENTED** [review]
one question

File: src/OSPSuite.Presentation/Presenters/Importer/ImportConfirmationPresenter.cs [hunk 2]
```diff
@@ -44,20 +60,21 @@ public void TriggerNamingConventionChanged(string namingConvention)
       public void SetKeys(IEnumerable<string> keys)
       {
          View.SetNamingConventionKeys(keys);
+         _keys = keys;
       }
 
-      public void SetNamingConventions (IEnumerable<string> namingConventions)
+      public void SetNamingConventions(IEnumerable<string> namingConventions)
```
Comment (msevestre) [inline src/OSPSuite.Presentation/Presenters/Importer/ImportConfirmationPresenter.cs:66]: can that be IReadOnly

File: src/OSPSuite.Presentation/Presenters/Importer/ImportConfirmationPresenter.cs [hunk 3]
```diff
@@ -44,20 +60,21 @@ public void TriggerNamingConventionChanged(string namingConvention)
       public void SetKeys(IEnumerable<string> keys)
```
Comment (msevestre) [inline src/OSPSuite.Presentation/Presenters/Importer/ImportConfirmationPresenter.cs:60]: can that be IReadOnly

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**